src/gasman.c

```diff
@@ -1325,7 +1339,8 @@ UInt ResizeBag (
         /* check that enough storage for the new bag is available          */
         if ( SizeAllocationArea <  WORDS_BAG(sizeof(BagHeader)+new_size)
               && CollectBags( new_size, 0 ) == 0 ) {
-            return 0;
+            fputs("gap: cannot extend the workspace any more!!!!!!\n",stderr);
+            SyExit( 1 );
```

Comment:
Perhaps use `SyAbortBags` (not terribly important, just sayin')

Reply:
Ah, I didn't notice that function.